br/tests/lightning_duplicate_resolution_replace/data/dup_resolve.a.1.csv

```diff
@@ -2,3 +2,5 @@ a,b,c
 1,6,1.csv
 2,6,2.csv
 3,3,3.csv
+3,4,4.csv
+5,4,5.csv
```

Comment:
the new rows should also be checked in run.sh

Reply:
row 4 was deleted because of conflict with row 3, and row 5 was deleted because of conflict with row 4, so they should not appear in `dup_resolve.a`